PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalInternalClient.kt

```diff
@@ -131,6 +141,110 @@ internal class PayPalInternalClient(
             ?: redirectUri.getQueryParameter("token")
     }
 
+    @ExperimentalBetaApi
+    fun sendVaultEditRequest(
+        context: Context,
+        payPalVaultEditRequest: PayPalVaultEditRequest,
+        callback: PayPalVaultEditCallback
+    ) {
+        getClientMetadataId(
+            context
+        ) { clientMetadataId ->
+            if (clientMetadataId == null) {
+                val result = PayPalVaultEditResponse.Failure(
+                    BraintreeException("Could not retrieve clientMetaDataId")
```

Comment:
I'm not sure we want to expose this internal info to the merchant - this might be a case for "An unexpected error occurred"